PowerEditor/installer/nativeLang/german.xml

```diff
@@ -152,7 +153,7 @@
 					<Item id="42070" name="Gemischter &amp;Wortstil"/>
 					<Item id="42071" name="iNVERTIERTER wORTSTIL"/>
 					<Item id="42072" name="zUFäLlIGeR WortSTiL"/>
-					<Item id="42073" name="Als Datei öffnen"/>
+					<Item id="42073" name="Datei öffnen"/>
```

Comment:
Translation is heavily misleading, as it basically means nothing as whereas the current translation makes it clear, that N++ tries to open the select as a file.